src/app/components/PortraitVideoModal/index.styles.tsx

```diff
@@ -88,6 +88,45 @@ const styles = {
         },
       },
     }),
+
+  visuallyHiddenCloseButton:
+    (dir: 'ltr' | 'rtl' = 'ltr') =>
+    (theme: Theme) =>
+      css({
+        position: 'absolute',
+        top: `${theme.spacings.DOUBLE}rem`,
+        left: dir === 'ltr' ? `${theme.spacings.DOUBLE}rem` : undefined,
+        right: dir === 'rtl' ? `${theme.spacings.DOUBLE}rem` : undefined,
```

Comment:
These can be replaced with the direction-aware `insetInlineStart` property, like:
`insetInlineStart: `${theme.spacings.DOUBLE}rem`,`

Reply:
Thanks, I'll try that now

Reply:
done